### PR TITLE
splitting/mirroring article data to rtdb

### PR DESCRIPTION
**Pull request by scuffed-developer**
- update : adding article now adds data to rtdb too (lite version)
- fix : article thumbnail now is added to the rtdb article too.

### CODE REVIEW

**Comment by arunstop** [comment]
Will review your code soon! I think after this we need to show the article with the mirrored data from rtdb.

Also the firebaseApi file is getting bigger, i think we need to modularize that.

File: utils/services/network/FirebaseApi.ts [hunk 1]
```diff
@@ -325,6 +330,13 @@ export async function fbArticleAdd({
         // create new article with newly added thumbnail url
         const articleWithThumbnail = { ...article, thumbnail: thumbnailUrl };
 
+        // add some part of article to rtdb for searching purpose
+        try {
```
Comment (arunstop) [inline utils/services/network/FirebaseApi.ts:334]: can make a function out of this. Then call it

File: utils/services/network/FirebaseApi.ts [hunk 2]
```diff
@@ -352,6 +364,13 @@ export async function fbArticleAdd({
       }
     }
 
+    // add some part of article to rtdb for searching purpose
+    try {
```
Comment (arunstop) [inline utils/services/network/FirebaseApi.ts:368]: This too

File: utils/services/network/FirebaseApi.ts (hunk 1 above)
Comment (scuffed-developer) [reply utils/services/network/FirebaseApi.ts:334]: Will do!

File: utils/services/network/FirebaseApi.ts (hunk 2 above)
Comment (scuffed-developer) [reply utils/services/network/FirebaseApi.ts:368]: Okay

**Review by scuffed-developer — COMMENTED** [review]
Great!

**Review by arunstop — COMMENTED** [review]
Nice la!

**Review by arunstop — APPROVED** [review]

**Pull request merged**